AssetEditor/AssetEditor.csproj

```diff
@@ -4,7 +4,7 @@
         <TargetFramework>net6.0-windows</TargetFramework>
         <UseWPF>true</UseWPF>
         <ApplicationIcon>hnet.com-image (1).ico</ApplicationIcon>
-        <AssemblyName>AssetEditor v0.45</AssemblyName>
+        <AssemblyName>AssetEditor45-PhazerEdition</AssemblyName>
```

Comment:
It's better idea to call it AssetEditor v.05 instead to avoid any confusion with old userbase. Also for historical reason.

Reply:
also we can add our own credit in about window

Reply:
AssetEditor-Amiral-Phazer editor

Reply:
Just leave it AssetEditor v0.5, we can build our about dialog with our names and credit on it.
[Resolve this and then I can merge it]

Reply:
AssEd45 (version 0.2.1.1)